spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -156,7 +156,7 @@ module MiqAeServiceSpec
       before { NotificationType.seed }
 
       let(:options) { {} }
-      let(:workspace) { double("MiqAeEngine::MiqAeWorkspaceRuntime", :root => options) }
+      let(:workspace) { double("MiqAeEngine::MiqAeWorkspaceRuntime", :root => options, :ae_user => user) }
```

Comment:
@mkanoor - I have a separate PR for test cleanup:
https://github.com/ManageIQ/manageiq/pull/12317
Would you prefer I do that in this PR?